CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Extrahorizon CLI changelog
 
+### vx.x.x
```

Comment:
@chad-pelly What will the version be? We should still do the version upgrade ticket?